pkgs/by-name/en/entropy/package.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "entropy";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "EwenQuim";
+    repo = "entropy";
+    rev = "v${version}";
+    hash = "sha256-GFFBZMwGV+dnf05lWtUVFIeq7QXc3ylQn27OXBv9ND0=";
+  };
+
+  vendorHash = "sha256-rELkSYwqfMFX++w6e7/7suzPaB91GhbqFsLaYCeeIm4=";
+
+  meta = {
+    description = "Entropy is a CLI tool that will scan your codebase for high entropy lines, which are often secrets.";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs/#meta-attributes
```suggestion
    description = "Tool to scan your codebase for high entropy lines, which are often secrets";
```